apps/code-examples/src/app/code-examples/forms/input-box/basic/demo.component.html

```diff
@@ -81,6 +91,13 @@ <h2>New member form</h2>
               {{ colorError.message }}
             </sky-status-indicator>
           </div>
+          <!-- Custom form error not handled by input box. -->
```

Comment:
I think we should just remove the custom error above this one--we'll probably get rid of that way eventually, and all other components will just use sky-form-error